conf/layer.conf

```diff
@@ -13,3 +13,5 @@ LAYERDEPENDS_debian-extended = "core debian"
 LAYERSERIES_COMPAT_debian-extended = "warrior"
 
 LAYERDIR_DEBIAN_debian-extended = '${@os.path.normpath("${LAYERDIR}")}'
+
+LICENSE_PATH += "${LAYERDIR}/custom-licenses"
```

Comment:
Why don't we use `licenses` for the directory name like meta-oe?

Reply:
> Why don't we use `licenses` for the directory name like meta-oe?

It's exactly what you said.   I changed directory name to licenses.

Reply:
Confirmed. Thanks!